packages/myst-cli/src/process/mdast.ts

```diff
@@ -215,6 +219,19 @@ export async function transformMdast(
   // Combine file-specific citation renderers with project renderers from bib files
   const fileCitationRenderer = combineCitationRenderers(cache, ...rendererFiles);
 
+  if (execute) {
+    const cachePath = path.join(session.buildPath(), 'execute');
+    await kernelExecutionTransform(mdast, vfile, {
+      cache: new LocalDiskCache<(IExpressionResult | IOutput[])[]>(cachePath),
+      sessionFactory: () => session.jupyterSessionManager(),
+      frontmatter: frontmatter,
+      ignoreCache: false,
+      errorIsFatal: false,
+      log: session.log,
+    });
+  }
+  transformRenderInlineExpressions(mdast, vfile);
+
   transformFilterOutputStreams(mdast, vfile, frontmatter.settings);
   await transformOutputsToCache(session, mdast, kind, { minifyMaxCharacters });
```

Comment:
@rowanc1 this function now maps `output.data` from raw MIME to minified.

Reply:
Awesome!!

Reply:
Sorry @rowanc1, just to clarify -- the notebook ingest (`loadFile`) now loads the outputs in un-minified. We later re-write these to minified in the `transformOutputsToCache` on L236. So, I've not added any minification, I've just disabled the early-return that was previously there.

Reply:
@rowanc1 could you clarify your expectations? 

To my knowledge, the final AST _is_ minified, we just do it in a single place much later in the AST transform.

Reply:
Ok, that is my expectation. So hopefully all is fine! I saw that we were _maybe_ not passing an option somewhere (in a previous diff). I will test it out on a bunch of content and see if there are any issues!